docs/development/generate_blob.md

```diff
@@ -0,0 +1,13 @@
+# Generate blob for tests
+
+If you need to generate a blob for your test, to lock the behavior, there is the process:
+
+- Generate what you need (expected data)
+- Dump it (don't keep at the end)
+- Print it in hexadecimal format (don't keep at the end)
+- Use `hex!` macro and store the value in a variable, each row should contains `74` characters maximum
```

Comment:
Why the limit on `74` ? every other tools (hexdump, xxd) that print hex data do it in a multiple of $2^n$

Reply:
Because @touilleMan script generated 80 characters long:
- 74 for hexadecimal per row
- 2 for quote
- 4 for the indentation due to hex!

Reply:
In a way, I could use `64` since $64 < 74$ :smil